x-pack/legacy/plugins/siem/public/pages/detection_engine/signals/index.tsx

```diff
@@ -4,94 +4,294 @@
+          status,
+          setEventsDeleted: setEventsDeletedCallback,
+          setEventsLoading: setEventsLoadingCallback,
+          kbnVersion,
+        });
+      },
+      [selectedEventIds, setEventsDeletedCallback, setEventsLoadingCallback]
+    );
+    const sendSignalsToTimelineCallback: SendSignalsToTimeline = useCallback(async () => {
+      // TODO: Update data to use all signals
+      await sendSignalsToTimelineAction({
+        createTimeline: createTimelineCallback,
+        data: Object.values(selectedEventIds)[0],
+      });
+    }, [selectedEventIds, setEventsDeletedCallback, setEventsLoadingCallback]);
+
+    // Callback for creating the SignalUtilityBar which receives totalCount from EventsViewer component
+    const utilityBarCallback = useCallback(
+      (totalCount: number) => {
+        return (
+          <SignalsUtilityBar
+            areEventsLoading={loadingEventIds.length > 0}
+            clearSelection={clearSelectionCallback}
+            isFilteredToOpen={filterGroup === FILTER_OPEN}
+            selectAll={selectAllCallback}
+            selectedEventIds={selectedEventIds}
+            sendSignalsToTimeline={sendSignalsToTimelineCallback}
+            showClearSelection={showClearSelectionAction}
+            totalCount={totalCount}
+            updateSignalsStatus={updateSignalsStatusCallback}
+          />
+        );
+      },
+      [
+        clearSelectionCallback,
+        filterGroup,
+        loadingEventIds.length,
+        selectAllCallback,
+        selectedEventIds,
+        showClearSelectionAction,
+      ]
+    );
+
+    // Send to Timeline / Update Signal Status Actions for each table row
+    const additionalActions = useMemo(
+      () =>
+        getSignalsActions({
+          createTimeline: createTimelineCallback,
+          setEventsLoading: setEventsLoadingCallback,
+          setEventsDeleted: setEventsDeletedCallback,
+          status: filterGroup === FILTER_OPEN ? FILTER_CLOSED : FILTER_OPEN,
+          kbnVersion,
+        }),
+      [createTimelineCallback, filterGroup, kbnVersion]
+    );
 
     return (
-      <EuiFilterGroup>
-        <EuiFilterButton
-          hasActiveFilters={filterGroup === FILTER_OPEN}
-          onClick={() => {
-            setFilterGroup(FILTER_OPEN);
-            onFilterGroupChanged(FILTER_OPEN);
-          }}
-          withNext
-        >
-          {'Open signals'}
-        </EuiFilterButton>
-
-        <EuiFilterButton
-          hasActiveFilters={filterGroup === FILTER_CLOSED}
-          onClick={() => {
-            setFilterGroup(FILTER_CLOSED);
-            onFilterGroupChanged(FILTER_CLOSED);
-          }}
-        >
-          {'Closed signals'}
-        </EuiFilterButton>
-      </EuiFilterGroup>
+      <>
+        <GlobalTime>
+          {({ to, from }) => (
+            <StatefulEventsViewer
+              defaultIndices={['.siem-signals-spong-default']} // TODO Get from new FrankInspired XavierHook
```

Comment:
Need to revert this to the default until switching over to use the API to fetch this.